src/Server/featureFlags/unleashHelpers.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  NODE_ENV,
+  UNLEASH_API,
+  UNLEASH_APP_NAME,
+  UNLEASH_SERVER_KEY,
+} from "Server/config"
+import { Unleash, startUnleash } from "unleash-client"
+
+export function getOrInitUnleashServerClient(): Promise<Unleash> {
+  const config = {
+    url: UNLEASH_API,
+    appName: UNLEASH_APP_NAME,
+    refreshInterval: 10000,
+    environment: NODE_ENV,
+    customHeaders: {
+      Authorization: UNLEASH_SERVER_KEY,
+    },
+  }
+
+  return startUnleash(config)
```

Comment:
This will only ever return a single instance of the `Unleash` client since we're using `startUnleash`, [which implements a singleton](https://github.com/Unleash/unleash-client-node/blob/main/src/index.ts#L16-L30). I'll monitor this in staging, though.

Reply:
It was a bit unclear in the example given in the PR description but if this function returns a promise (as startUnleash does) – I imagine callers will need to wait on that result and handle any errors thrown, as documented [here](https://docs.getunleash.io/reference/sdks/node#synchronous-initialization).

```ts
import { getOrInitUnleashServerClient } from "Server/featureFlags/unleashHelpers"

// Get (or initialize) Unleash client
try {
  const unleash = await getOrInitUnleashServerClient()
} catch (e) {
  // handle error
}
  
```